src/commands/wait_for_job.yml

```diff
@@ -1,12 +1,12 @@
 description: |
-  This command waits for another job in this workflow. Since right now Circle CI doesnt let
+  This command waits for a job(s) in this workflow. Since right now Circle CI doesn't let
   you queue up jobs irrespective of whether they fail or not. This is a faux queue where
   the command stalls till the other job succeeds.
   This requires a Circle CI token be set as $CIRCLE_TOKEN
 
 parameters:
-  job-name:
-    description: The job on which to wait. If job not found continue immediately
+  job-names:
```

Comment:
@roopakv Instead of adding a new command, should we just keep it the same?

I've tested the code which should allow you to just pass in one job name and it will wait until that job is done. OR someone can pass in a list of job names comma separated and it will wait unit all jobs are done.

Reply:
yeah lets just keep as job-name and update the documentation to say we support multiple jobs :) makes life better.